src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -94,15 +94,16 @@ std::string LiveFunctionsDataView::GetValue(int row, int column) {
     return "";
   }
 
-  const uint64_t function_id = GetInstrumentedFunctionId(row);
-  const ScopeStats& stats = app_->GetCaptureData().GetScopeStatsOrDefault(function_id);
+  const uint64_t scope_id = GetScopeId(row);
+  const ScopeStats& stats = app_->GetCaptureData().GetScopeStatsOrDefault(scope_id);
 
-  const FunctionInfo& function = *GetFunctionInfoFromRow(row);
+  const FunctionInfo* function = GetFunctionInfoFromRow(row);
   switch (column) {
     case kColumnSelected:
-      return FunctionsDataView::BuildSelectedColumnsString(app_, function);
+      return function == nullptr ? ""
```

Comment:
Is it possible that the `function` is `nullptr` here?   
If not, I prefer to use `ORBIT_CHECK(function != nullptr)` before the `switch` instead of adding the `nullptr` in `kColumnSelected`, `kColumnModule` and `kColumnAddress` cases.

Reply:
Never mind, just notice that this can be null on manual instrumentation.